executor/utils.py

```diff
@@ -109,13 +110,23 @@ def get_user_agent():
     }
 
 
-def upload_file(client, report_name, report_id):
+def upload_file(client, report_name, report_id, owner_id):
     report_filename = os.path.basename(report_name)
-    return client.ns('reporting').reports[report_id].action('upload').post(
-        data=open(report_name, 'rb'),
+    response = requests.post(
```

Comment:
why not using the client??? you can do the same without requests :)

Reply:
I tried and wanted.... but broke my head trying to figure out the final expression

Reply:
Like you get` client.ns('media').folders['reports_report_file'] ` and then I'm stuck

Reply:
@bdjilka try:

`client.ns('media').ns('folders').collection('reports_report_file')[owner_id].action('files').post(files=...., headers=...)`

And send only content-type and content-disposition headers.